src/NosCore.Database/Entities/NpcMonster.cs

```diff
@@ -87,8 +87,6 @@ public NpcMonster()
 
         public byte HeroLevel { get; set; }
 
-        public int HeroXp { get; set; }
```

Comment:
On official, there is no data about that in the monster informations, it's working like that :
If a monster is in act 6 - 7 so he can give u some hxp (monsterXp / 25)

Reply:
this is still calculation that should happen on parser time to make it more efficient

Reply:
just set HXP to monsterXp/25